crt/glibc_stub.c

```diff
@@ -62,6 +64,33 @@ int __argz_create_sep(const char* string, int delim, char** argz, size_t* len)
 }
 weak_alias(__argz_create_sep, argz_create_sep);
 
+error_t __argz_append(
+    char** argz,
+    size_t* argz_len,
+    const char* buf,
+    size_t buf_len)
+{
+    assert(0);
+    return 0;
```

Comment:
0 rerun indicates success.
```suggestion
    return ENOMEM;
```
Also on line 60, to be consistent, and for readability, suggest to use `error_t` return type.